staging/src/k8s.io/apiserver/pkg/util/flowcontrol/default-objects.go

```diff
@@ -0,0 +1,193 @@
+				users(user.APIServerUser),
+				groups(user.SystemPrivilegedGroup)...,
+			),
+			Rule: nonResourceRule(
+				[]string{rmv1a1.VerbAll},
+				[]string{rmv1a1.NonResourceAll},
+			)},
+	)
+)
+
+// DefaultPriorityLevelConfigurationObjects returns the instances of
+// the ones defined in the KEP, with system-top in position 0 and
+// workload-low in position 1
+func DefaultPriorityLevelConfigurations() []*rmv1a1.PriorityLevelConfiguration {
+	return []*rmv1a1.PriorityLevelConfiguration{
+		DefaultPriorityLevelConfigurationExempt,
+		pl("system-high", rmv1a1.PriorityLevelConfigurationSpec{
+			AssuredConcurrencyShares: 100,
+			Queues:                   128,
+			HandSize:                 6,
+			QueueLengthLimit:         10}),
+		pl("system-low", rmv1a1.PriorityLevelConfigurationSpec{
+			AssuredConcurrencyShares: 30,
+			Queues:                   1,
+			QueueLengthLimit:         1000}),
+		pl("workload-high", rmv1a1.PriorityLevelConfigurationSpec{
+			AssuredConcurrencyShares: 30,
+			Queues:                   128,
+			HandSize:                 6,
+			QueueLengthLimit:         100}),
+		DefaultPriorityLevelConfigurationExempt,
+	}
+}
+
+// DefaultFlowSchemaObjects returns instances of the FlowSchema
+// objects shown in the Example Configuration section of the KEP
+func DefaultFlowSchemas() []*rmv1a1.FlowSchema {
+	verbAll := []string{rmv1a1.VerbAll}
+	apiGroupAll := []string{rmv1a1.APIGroupAll}
+	apiGroupCore := ""
+	resourceAll := []string{rmv1a1.ResourceAll}
+	ruleRscAll := resourceRule(verbAll, apiGroupAll, resourceAll)
+	ruleNonRscAll := rmv1a1.PolicyRule{
+		Verbs:           verbAll,
+		NonResourceURLs: []string{rmv1a1.NonResourceAll}}
+	subjectsAll := groups(user.AllAuthenticated, user.AllUnauthenticated)
+	return []*rmv1a1.FlowSchema{
+		DefaultFlowSchemaSystemTop,
+		fs("system-high", "system-high", 2500,
+			rmv1a1.FlowDistinguisherMethodByUserType,
+			rmv1a1.PolicyRuleWithSubjects{
+				Subjects: groups(user.NodesGroup),
+				Rule:     resourceRule(verbAll, []string{apiGroupCore}, []string{"nodes"})},
+			rmv1a1.PolicyRuleWithSubjects{
+				Subjects: groups(user.NodesGroup),
+				Rule:     ruleRscAll},
+			rmv1a1.PolicyRuleWithSubjects{
+				Subjects: users(user.KubeControllerManager, user.KubeScheduler),
+				Rule:     resourceRule(verbAll, []string{apiGroupCore}, []string{"endpoints", "configmaps", "leases"})},
+			rmv1a1.PolicyRuleWithSubjects{
+				Subjects: users(user.KubeControllerManager, user.KubeScheduler),
+				Rule:     ruleNonRscAll},
+		),
+		fs("system-low", "system-low", 3500,
+			rmv1a1.FlowDistinguisherMethodByUserType,
+			rmv1a1.PolicyRuleWithSubjects{
+				Subjects: kubeSystemServiceAccount("generic-garbage-collector"),
+				Rule:     ruleRscAll,
+			},
+			rmv1a1.PolicyRuleWithSubjects{
+				Subjects: kubeSystemServiceAccount("generic-garbage-collector"),
+				Rule:     nonResourceRule([]string{rmv1a1.VerbAll}, []string{rmv1a1.NonResourceAll}),
+			},
+		),
+		fs("workload-high", "workload-high", 7500,
```

Comment:
This matches everything.  Is that the reason there is no "workload-low"?
The KEP calls for a distinction between the two, but making that expressable will require some additional work that can be done in later PRs.